### PR TITLE
add esc key to close paste modal

### LINKED ISSUE #16: Esc doesn't cancel the paste.
Nothing happens when the ESC key pressed while in the paste mode. 

Hyper version 3.0.2
Windows 10

### PR DESCRIPTION
**Pull request by trystan2k**
Add escape key to close paste modal

Closes #16

### CODE REVIEW

**Review by zsol — APPROVED** [review]
yep, looks like code :)

File: src/PasteDialog.react.js [hunk 1]
```diff
@@ -1,10 +1,12 @@
+/* eslint-disable no-console */
```
Comment (zsol) [inline src/PasteDialog.react.js]: maybe remove this though

Comment (trystan2k) [reply src/PasteDialog.react.js]: Sorry, forgot to remove. It is done now.

I will merge the PR, can you please do the release, when you got some time ? Thanks @zsol !

Comment (trystan2k) [reply src/PasteDialog.react.js]: It is merged @zsol . Could you please do the release ?

Thanks !

**Pull request merged**